EU4ToVic2Tests/MapperTests/IdeaEffectsTests.cpp

```diff
@@ -35,20 +35,20 @@ TEST(Mappers_IdeaEffectsTests, armyInvestmentValueDefaultsToZero)
 
 	mappers::IdeaEffects ideaEffects(input);
 
-	ASSERT_EQ(ideaEffects.getArmyInvestmentValue(), 0);
+	ASSERT_EQ(ideaEffects.getArmy(), 5);
```

Comment:
Test should be renamed to match. Same for other modified tests.

Reply:
all tests rewritten and renamed.